api/utils.py

```diff
@@ -93,6 +94,7 @@ def build_flask_app(name):
         app = Flask(name, static_url_path="", static_folder="frontend/public")
 
     ExceptionHandler(app, running_local=running_local())
+    SecurityHeaders(app, running_local=running_local())
```

Comment:
suggestion: can the `CORS` management moved inside `SecurityHeaders`? There's already a check for running local, and by doing that we'll end up having all security related stuff in the same place

Reply:
I like the idea, I've done it in the next rev coming up, but I had to keep a check. for the different strategy in building the flask app